app/scripts/components/common/mapbox/layers/raster-timeseries.tsx

```diff
@@ -273,13 +273,34 @@ export function MapLayerRasterTimeseries(props: MapLayerRasterTimeseriesProps) {
         LOG && console.groupEnd();
         /* eslint-enable no-console */
 
-        const responseData = await requestQuickCache({
-          url: `${tileApiEndpointToUse}/mosaic/register`,
-          payload,
-          controller
-        });
-
-        setMosaicUrl(responseData.links[1].href);
+        let responseData;
+
+        try {
+          responseData = await requestQuickCache({
+            url: `${tileApiEndpointToUse}/searches/register`,
+            payload,
+            controller
+          });
+          const mosaicUrl = responseData.links[1].href;
+          setMosaicUrl(mosaicUrl.replace('/{tileMatrixSetId}', '/WebMercatorQuad'));
+        } catch (e) {
+          // @NOTE: conditional logic TO BE REMOVED once new BE endpoints have moved to prod... Fallback on old request url if new endpoints error with nonexistance... 
+          if (e.request) {
+            // The request was made but no response was received
+            responseData = await requestQuickCache({
+              url: `${tileApiEndpointToUse}/mosaic/register`, // @NOTE: This will fail anyways with "staging-raster.delta-backend.com" because its already deprecated...
+              payload,
+              controller
+            });
+            
+            const mosaicUrl = responseData.links[1].href;
+            setMosaicUrl(mosaicUrl);
+          } else {
+              LOG && 
+              /* eslint-disable-next-line no-console */
+              console.log('Titiler /register endpoint error', 'color: red;', e);
```

Comment:
Should we change the status here aside from the logging? Otherwise it might not propagate the error to the UI in case something goes wrong.

`changeStatus({ status: S_FAILED, context: STATUS_KEY.Layer })`